Package lambdas as part of `make deploy` (#1997) (#2802)
DataBiosphere/azul: README.md

the shared cloud resources, it is time to provision the cloud infrastructure
for your deployment. Run

```
make deploy
```

> to prepare the lambda functions defined in the `lambdas` directory for

Reviewer: ```suggestion
to prepare the Lambda functions defined in the `lambdas` directory for
```

deployment via Terraform. It will display a plan and ask you to confirm it.
Please consult the Terraform documentation for details.

Any time you wish to change the code running in the lambdas you will need to
run `make deploy`.

> Some Terraform configuration is generated by `make -C lambdas`, but the rest is
> defined in `….tf.json` files which in turn are generated from
> `….tf.json.template.py` files which are simple Python scripts containing the
> desired JSON as Python dictionary and list literals and comprehensions. Changes
> to these files require running `make deploy` again in order to update
> cloud infrastructure for the selected deployment.

Reviewer: ```suggestion
Some Terraform configuration is generated by `make -C lambdas`, but the rest is
defined in `….tf.json` files which in turn are generated from
`….tf.json.template.py` templates which are simple Python scripts containing the
desired JSON as Python dictionary and list literals and comprehensions. `make deploy` will take care of both, `make -C lambda` as well as expanding the template files. Changes
to either the templates or anything in the `lambdas` directory requires running `make deploy` again in order to update
cloud infrastructure for the selected deployment.
```

> If you are provisioning multiple deployments, you will need to run `make deploy`
> for each one, since the Terraform configuration generated by `make -C lambdas`
> is deployment specific.

Reviewer: ```suggestion
```

That should be obvious now.
